prac_07/project_management.py

```diff
@@ -46,6 +48,7 @@ def load_project(file_name):
 
 
 def display_projects(projects):
+    """Display project"""
     incomplete_projects = [project for project in projects if project.completion < 100]
     completed_projects = [project for project in projects if project.completion == 100]
 
```

Comment:
The program does not print the list when I choose menu option D for display.

Reply:
its because the projects  aren’t loaded at the beginning of the program